stock-service/src/main/resources/application-dev.yml

```diff
@@ -56,6 +56,10 @@ jwt:
 alphavantage:
   api-key: ${ALPHAVANTAGE_APY_KEY}
 
+listings:
+  refetch-min: 20000000
+  # 2 for testing, 15 for PM, 20000000 for others
```

Comment:
please set this to 15 in `application.properties` (so that 15 is used in production)